crates/db_views_moderator/src/structs.rs

```diff
@@ -26,52 +26,52 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct ModAddCommunityView {
   pub mod_add_community: ModAddCommunity,
-  pub moderator: PersonSafe,
+  pub moderator: Option<PersonSafe>,
   pub community: CommunitySafe,
   pub modded_person: PersonSafeAlias1,
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct ModAddView {
   pub mod_add: ModAdd,
-  pub moderator: PersonSafe,
+  pub moderator: Option<PersonSafe>,
```

Comment:
This is a minor regression, because the mod log currently shows whether a specific action was taken by a community mod or by a site admin. That wont be possible anymore. But fixing that seems extremely complicated and not worth the trouble, so lets leave it like this.